recipes/default.rb

```diff
@@ -100,19 +100,21 @@
     end
   end
 
-  hostsfile_entry 'localhost' do
-    ip_address '127.0.0.1'
-    hostname 'localhost'
-    action :append
-  end
+  if node['hostname_cookbook']['manage_hostfile']
```

Comment:
I've [cut a new release](https://github.com/3ofcoins/chef-cookbook-hostname/releases/tag/v0.4.1), and would be happy to include this PR in another release if you wouldn't mind applying the requested changes.